core/iwasm/common/wasm_application.c

```diff
@@ -632,9 +631,7 @@ execute_func(WASMModuleInstanceCommon *module_inst, const char *name,
     if (argv1)
         wasm_runtime_free(argv1);
 
-    exception = wasm_runtime_get_exception(module_inst);
-    bh_assert(exception);
-    os_printf("%s\n", exception);
```

Comment:
Duplicated exception dump is added in iwasm main.c after calling app_instance_func.